static/bidder-params/conversant.json

```diff
@@ -1,12 +1,12 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "Conversant Adapter Params",
-  "description": "A schema which validates params accepted by the Conversant adapter.",
+  "title": "Epsilon (formerly Conversant) Adapter Params",
+  "description": "A schema which validates params accepted by the Epsilon (Conversant) adapter.",
   "type": "object",
   "properties": {
     "site_id": {
       "type": "string",
-      "description": "A Conversant specific ID which identifies the site."
+      "description": "An Epsilon (Conversant) specific ID which identifies the site."
```

Comment:
this info is related to conversant. Any specific reason to mention  epsilon here

Reply:
Conversant Media doesn't exist anymore, we've merged with Epsilon so this works is to try and start moving our branding in that direction.  However to avoid confusion, it is good to reference our legacy name until we are 100% sure we want to remove it completely.